scripts/add_metrics_to_readme.sh

```diff
@@ -9,6 +9,27 @@ function getMetrics() {
     done
 }
 
+function failureConfigChange() {
+	local logstashCID=$(docker ps -a | grep 'logstash-exporter-logstash-1' | awk '{print $1}')
+	local logstashPID=$( docker exec "$logstashCID" sh -c "echo \$(ps aux | grep logstash) | awk '{print \$2}'" )
+	local logstashConf='/usr/share/logstash/pipeline/logstash.conf'
+
+	docker exec -it "$logstashCID" sh -c "echo 'Wrong Config' >> $logstashConf"
+	# reload logstash
+	docker exec -it "$logstashCID" sh -c "kill -1 $logstashPID"
+
+	# bring back previous config 
+	# walk around with 'cp' to avoid replacing config inside container
+	docker exec "$logstashCID" sh -c "sed '\$d' $logstashConf > /tmp/prev_logstash.conf" 
+	docker exec "$logstashCID" sh -c "cp /tmp/prev_logstash.conf $logstashConf"
+
+	# reload logstash
+	docker exec -it "$logstashCID" sh -c "kill -1 $logstashPID"
```

Comment:
logstash pid should be always 1 AFAIK

Reply:
Okey, I will change it, but it's on you my friend if it doesn't work.